test/array.specs.ts

```diff
@@ -0,0 +1,52 @@
+import "mocha";
+import fc from "fast-check";
+import { last } from "../src/array";
+import { strict as assert } from "assert";
+
+
+describe("last: ", () => {
+  it("Should return the last element of a non-empty array", () => {
+    fc.assert(
+      fc.property(fc.array(fc.integer(), { minLength: 1 }), (arr) => {
+        const result = last(arr);
+        return result === arr[arr.length - 1];
+      })
+    );
+  });
+
+
+    it("Should return undefined for an empty array", () => {
+      fc.assert(
+        fc.property(fc.array(fc.anything(), { maxLength: 0 }), (arr) => {
+          assert.strictEqual(last(arr), undefined);
+        })
+      );
+    });
+
+  it("Should handle arrays of any type", () => {
+    fc.assert(
+      fc.property(fc.array(fc.anything(), { minLength: 1 }), (arr) => {
+        const result = last(arr);
+        return result === arr[arr.length - 1];
+      })
+    )
+  })
+
+  it("Should return the only element for a single-element array", () => {
+    fc.assert(
+      fc.property(fc.anything(), (element) => {
+        const arr = [element];
+        assert.strictEqual(last(arr), element);
+      })
+    );
+  });
+
+  it("Should return the last element in a reversed array", () => {
```

Comment:
I dont think this test is necessary, it just means that reverse and last are working in there definition, but this is already done above